### PR TITLE
Update stringtie default parameter values

### PR DESCRIPTION
**Pull request by gallardoalba**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

Currently StringTie uses as default -f parameter a value of 0.15, but it implies to lose a lot of information by removing relative low abundance isoforms, even if sometimes the transcripts that are filtered out are highly expressed. The default value according the developers should be 0.01.

### CODE REVIEW

**Review by bgruening — APPROVED** [review]

**Comment by bernt-matthias** [comment]
Could you check also the other parameters 

I just had a quick look in the stringtie sources and have seen that `-c` is documented to use a default of 1 and has a default of  2 in the wrapper...

**Comment by gallardoalba** [comment]
> > Could you check also the other parameters
> > I just had a quick look in the stringtie sources and have seen that `-c` is documented to use a default of 1 and has a default of 2 in the wrapper...
> 
> Sure, I checked a few of them, but it seems that I missed some.



> Could you check also the other parameters
> 
> I just had a quick look in the stringtie sources and have seen that `-c` is documented to use a default of 1 and has a default of 2 in the wrapper...

Also the parameter -M was different with respect to the default parameters. I think all of them are fine now.

File: tools/stringtie/stringtie.xml [hunk 1]
```diff
@@ -417,7 +417,13 @@ $adv.multi_mapping
                 <param name="input_bam_short" ftype="bam" value="short_reads.bam" />
                 <param name="input_bam_long" ftype="bam" value="long_reads.bam" />
             </conditional>            
-            <output name="output_gtf" file="stringtie_out12.gtf" ftype="gtf" compare="re_match" sort="true"/>
+            <output name="output_gtf" ftype="gtf">
+                <assert_contents>
+                    <has_text text="5.043160"/>
+                    <has_text text="StringTie version 2.2.1"/>
```
Comment (bernt-matthias) [inline tools/stringtie/stringtie.xml:423]: You might want to use the `@TOOL_VERSION@` token here?

**Review by bernt-matthias — APPROVED** [review]

File: tools/stringtie/stringtie.xml (hunk 1 above)
Comment (gallardoalba) [inline tools/stringtie/stringtie.xml:423]:
```suggestion
                    <has_text text="StringTie version @TOOL_VERSION@"/>
```

**Pull request merged**